Makefile

```diff
@@ -235,21 +235,11 @@ test-cmd-devfile-exec:
 test-cmd-docker-devfile-exec:
 	ginkgo $(GINKGO_FLAGS) -focus="odo docker devfile exec command tests" tests/integration/devfile/docker
 
-# Run odo status devfile command tests
-.PHONY: test-cmd-devfile-status
-test-cmd-devfile-status:
-	ginkgo $(GINKGO_FLAGS) -focus="odo devfile status command tests" tests/integration/devfile/
-
```

Comment:
why are we removing the status integration tests?

Reply:
I am not sure why it is marked for deletion. My change was to add test-e2e-devfile .

Could it be that I forgot to run a checkout from master before starting my changes?

Reply:
The app and status tests were newly added and maybe that's why they are shown removed.